tests/text/modules/heads/test_token_classification.py

```diff
@@ -60,6 +60,8 @@ def trainer_dict() -> Dict:
 def test_train(pipeline_dict, training_data_source, trainer_dict, tmp_path):
     pipeline = Pipeline.from_config(pipeline_dict)
 
+    assert pipeline.output == ["entities", "labels"]
```

Comment:
This is a bit confusing. Right now the `pipeline.output` are the optional arguments of the `featurize` method of the head. But the `pipeline.predict` does not return a `labels`, but a `tags` ... maybe we should somehow enforce a consistency between the two things, or better define what `pipeline.output` actually means.

Reply:
For now, I've updated the `featurize` method, renaming `labels` to `tags`.   

We must define a data model for task input/outputs that will keep this logic, reducing this kind of frictions/confussions.